### ISSUE TITLE
[Bug Report] "File exists" error after deleted in system file manager 

### ISSUE THREAD

**Issue by thepaleguy** [JunkFood02/Seal#358]
### Describe the bug

it says the file exists, but i deleted the file on my Manager

### Error reports & Screenshots

![Screenshot_20221120-232938_Seal](https://user-images.githubusercontent.com/96138215/202910922-4d027531-a686-4c4e-8f20-e50f84affbb0.jpg)

**Comment by JunkFood02** [comment]
This is an known issue which also occurs to me. I've still got no idea about how to solve this.

**Comment by Marco-9456** [comment]
> This is an known issue which also occurs to me. I've still got no idea about how to solve this.

I use the `Clear temporary files` option as a workaround.

**Comment by thepaleguy** [comment]
> > This is an known issue which also occurs to me. I've still got no idea about how to solve this.
> 
> I use the `Clear temporary files` option as a workaround.

clearing temp files also not working
this one is different issue

**Comment by JunkFood02** [comment]
Yes, this issue occurs in special occasions. As for me, the file needs to be deleted directly in the system's file manager (instead of a third party one). And this problem can only be reproduced on my MIUI devices, so I guess the issue is on the file manager's side.
